webapp/WEB-INF/templates/admin/system/manage_caches.html

```diff
@@ -7,6 +7,10 @@
 					<@button type='submit' buttonIcon='refresh' title='#i18n{portal.system.manage_caches.titleReloadProperties}' id='reload' size='' showTitleXs=false showTitleSm=false />
 					<@aButton href='jsp/admin/system/CacheInfos.jsp' buttonIcon='key' title='#i18n{portal.system.manage_caches.buttonViewKeys}' size='' showTitleXs=false showTitleSm=false />
 				</@tform>
+				<@tform method='post' action='jsp/admin/system/DoReloadLogsConfiguration.jsp' class='pull-right spaced'>
+					<input type="hidden" name="token" value="${token}">
+					<@button type='submit' buttonIcon='refresh' title='#i18n{portal.system.manage_caches.titleReloadLogsConfiguration}' id='reloadLogsConf' size='' showTitleXs=false showTitleSm=false />

```

Comment:
So the button shows no text on small screens ? Since there are no tooltips on smartphone, the purpose of this button might not be obvious

Reply:
If we go with having this button on this page, then maybe we can use a single letter to disambiguate ? L and P. Or Logs and Props ? Or Logs and Configuration ?
If we put this functionality in another page maybe the "reload" glyph is enough

Reply:
why not just display the title at all sizes ?